### PR TITLE
chore: updating dependabot to support TS, JS and docker packages.

### PR DESCRIPTION
**Pull request by rajpalc7**

### CODE REVIEW

**Comment by rajpalc7** [comment]
Hi @wadeBarnes, this is ready for review now.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
@rajpalc7, You're missing a supported `package-ecosystem`.

**Comment by rajpalc7** [comment]
Thanks @WadeBarnes , Its added now.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  - package-ecosystem: "npm"
+    directory: "/TractionIssuanceDemo"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major", "version-update:semver-patch"]
```
Comment (WadeBarnes) [inline .github/dependabot.yml]: @rajpalc7, You duplicated this section.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Found a duplicate section

File: .github/dependabot.yml (hunk 1 above)
Comment (rajpalc7) [reply .github/dependabot.yml]: Thanks @WadeBarnes duplication deleted now

**Review by WadeBarnes — APPROVED** [review]

**Pull request closed without merging**